### ISSUE TITLE
Raspberry Pi master and Arduino slave

### ISSUE THREAD

**Issue by SForney54** [ricaun/DuinoCoinI2C#28]
Just thoughts, it is possible to use Raspberry Pi as master and Arduino as slave through the RPI's GPIO?

**Comment by panayotovip** [comment]
It will be very nice if it can use I2C from Pi to add arduino nodes is serial.
Very interesting approach.

**Comment by SForney54** [comment]
Agreed, since RPI's I2C pins is on GPIO 2 and GPIO 3 which it allows for two-wire communication. I don't mind trying that approach but not sure where to start with. I'll be reviewing the examples of ESP8266/Arduino here and trying to get some idea from there.

**Comment by JK-Rolling** [comment]
it's available at https://github.com/JK-Rolling/DuinoCoinI2C_RPI
